digital-twins/jsonld-contexts/observed_attributes.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": {
+    "UseCase": "https://smartdatamodels/UseCase",
```

Comment:
SDM namespace is more like this: https://github.com/smart-data-models/dataModel.WaterQuality/blob/master/context.jsonld